mbed_greentea/mbed_greentea_cli.py

```diff
@@ -180,6 +183,45 @@ def filter_names_by_prefix(test_case_name_list, prefix_name):
 
     return filtered_ctest_test_list
 
+def create_testlink_test_list(test_list, available_test_cases, filtered_test_cases, test_spec):
+    """! Filters test case list (filtered with testlink test plan) and return filtered list.
+    @test_list Dict of Test Objects against their names
+    @available_test_cases List of test cases from test_spec
+    @filtered_test_cases All of the chosen test cases from test plan
+    @param test_spec Test specification object loaded with --test-spec switch
+    @return A set of tests to run, filtered by test cases
+    """
+
+    filtered_test_list = {}
+    invalid_test_cases = []
```

Comment:
Since this is list of test cases for which we couldn't find the binary it can be called ```not_implemented_tests```.